locales/fr/homepage.json

```
@@ -36,6 +36,9 @@
   "reuses.tosdr.author": "ToS;DR",
   "reuses.tosdr.subtitle": "Permet aux utilisateurs de donner collaborativement une note aux conditions d’utilisation selon le respect de leurs droits.",
   "reuses.tosdr.title": "Terms of Service; Didn’t Read",
+  "reuses.report-lab-lex.author": "Lab-LEX",
+  "reuses.report-lab-lex.title": "Gouvernance et régulation des réseaux sociaux",
+  "reuses.report-lab-lex.subtitle": "Étude exploratoire sur l’évolution des conditions d’utilisation des reseaux sociaux et leur impact sur les droits de l’homme.",
```

Please make sure to apply the changes on every translation before asking for a review 🙂 
```suggestion
  "reuses.report-lab-lex.author": "Lab-LEX, Université de Brest",
  "reuses.report-lab-lex.title": "Recherche sur l'impact sur les droits humains",
  "reuses.report-lab-lex.subtitle": "Papier de recherche sur l'impact sur les droits de l’homme de l’évolution des conditions d’utilisation des réseaux sociaux.",
```